### PR TITLE
WEB-2: Implement color system

### LINKED ISSUE #12: WEB-2: Implement color system
### Acceptance Criteria

1. Add colors to the theme in `tailwind.config.ts`.
    1. The colors can be seen in the Figma file.
2. Test the colors by applying the class to `className` in a `<p>` or header HTML tags.

As a reference, this is my `tailwind.config.ts` for my portfolio website. Note that you only need to implement the colors. **Note: Don’t copy my code exactly since it contains additional things you may not need. Use it as a reference and only look at the things that you think you will need.**

```tsx
import type { Config } from "tailwindcss";

const config: Config = {
  content: [
    "./src/pages/**/*.{js,ts,jsx,tsx,mdx}",
    "./src/components/**/*.{js,ts,jsx,tsx,mdx}",
    "./src/app/**/*.{js,ts,jsx,tsx,mdx}",
  ],
  darkMode: "selector",
  theme: {
    screens: {
      sm: "0px",
      md: "768px",
      lg: "1280px",
      xl: "1280px",
      "2xl": "1280px",
    },
    extend: {
      fontFamily: {
        cooperbt: ["var(--font-cooperbt)"],
        tthovespro: ["var(--font-tthovespro)"],
      },
      colors: {
        navbar: {
          l: "rgba(253, 249, 242, 0.48)",
          d: "rgba(34, 39, 36, 0.48)",
        },
        neutral: {
          100: "#F8F9FA",
          200: "#E9ECEF",
          300: "#DEE2E6",
          400: "#CED4DA",
          500: "#ADB5BD",
          600: "#6C757D",
          700: "#495057",
          800: "#343A40",
          900: "#212529",
        },
        primary: {
          100: "#EEF3F0",
          200: "#DEE7E0",
          300: "#CDDCD1",
          400: "#BDD0C1",
          500: "#ACC4B2",
          600: "#8A9D8E",
          700: "#67766B",
          800: "#454E47",
          900: "#222724",
        },
        secondary: {
          100: "#FEFDFB",
          200: "#FDFBF6",
          300: "#FDF9F2",
          400: "#FCF7ED",
          500: "#FBF5E9",
          600: "#C9C4BA",
          700: "#97938C",
          800: "#64625D",
          900: "#32312F",
        },
      },
    },
  },
  plugins: [],
};
export default config;

```

### Out of Scope

1. Any other design system components.

### PR DESCRIPTION
**Pull request by helenjb**
## Overview

Implemented color system.

## Changes Made

Added colors to `tailwind.config.ts`.

### CODE REVIEW

File: tailwind.config.ts
Comment (vinnie4k) [inline tailwind.config.ts]:
```suggestion
          black: "#000000",
```

Do you have the Prettier extension installed? It should automatically add a comma at the end.

File: tailwind.config.ts
Comment (vinnie4k) [inline tailwind.config.ts]:
```suggestion
          "offWhite": "#FAFAFA",
```
The dash is good, but the reason why we should remove it is that it's a bit confusing when using it with Tailwind since a dash is often used in class names. Also, the `.prettierrc` file should be adding a double-quote around everywhere. Double-check and make sure that you have the Prettier extension.

File: tailwind.config.ts
Comment (vinnie4k) [inline tailwind.config.ts]: Oops, forgot to remove this. Can you remove the `backgroundImage`?

**Review by vinnie4k — CHANGES_REQUESTED** [review]

File: tailwind.config.ts
Comment (helenjb) [reply tailwind.config.ts]: Yeah, I think my VS Code is buggy so the changes by formatting extensions aren't applied automatically, I'll fix it ASAP

File: tailwind.config.ts
Comment (vinnie4k) [inline tailwind.config.ts]: Missing a `#` in the front.

**Review by vinnie4k — APPROVED** [review]
Just one more comment. Once resolved, don't merge until I move the Notion ticket under "Ready to Merge."

**Pull request merged**
